.travis.yml

```diff
@@ -2,6 +2,8 @@ rvm:
   - 2.1.9
   - 2.2.4
   - 2.3.1
+  - 2.4.2
+  - 2.5.1
```

Comment:
Not sure about this: if we use major.minor (e.g. `2.5`), won't we test against the latest release of Ruby as soon as a newer `2.5.x` version is released?

Reply:
that would be based on the rvm version that's installed...  I'm not very familiar with travisci's config, and was just following the pattern of the previous entries.

Reply:
I'm not a travis expert neither.  Looking at other projects which use X.Y, I see these versions in a recent build:

| `.travis.yml` | Ruby version used | Latest ruby version available |
| 2.3 | 2.3.4 | 2.3.7 |
| 2.4 | 2.4.1 | 2.4.4 |
| 2.5 | 2.5.1 | 2.5.1 |

This look… inconsistent :smile:

Reply:
agreed.  I just noticed the same in the travis builds.  seems like I should leave the patch versions in place so we know what we're getting?

Reply:
:+1: